semcore/notice/src/Notice.jsx

```diff
@@ -36,6 +36,16 @@ class RootNotice extends Component {
     locale: 'en',
   };
 
+  constructor(props) {
+    super(props);
+
+    logger.warn(
```

Comment:
actually, in the task I meant writing about this in the docs, but this is also neat
but
this shouldn't fire for the "muted" theme, and probably for the "critical" either. I think it would be reasonable to only show this warning for the themes that have the same "Notification" aria-label.

Reply:
another option would be to keep showing this for all themes, but lower the level to info (or whatever is below warning)

Reply:
added check for themes